README.md

```diff
@@ -1,4 +1,105 @@
+
+### Overview
+
+The SACD JSON follows a consistent structure, as seen in the template:
+
+```json
+{
+  "specversion": "1.0",
+  "timestamp": "0000-00-00T00:00:00Z",
+  "type": "dimo.sacd",
+  "data": {
+    "grantor": {
+      "address": "0x0000000000000000000000000000000000000000",
+      "name": "",
+      "additionalInfo": {}
+    },
+    "grantee": {
+      "address": "0x0000000000000000000000000000000000000000",
+      "name": "",
+      "additionalInfo": {}
+    },
+    "effectiveAt": "0000-00-00T00:00:00Z",
+    "expiresAt": "0000-00-00T00:00:00Z",
+    "additionalDates": {},
+    "agreement": [
+      {
+        "type": "",
+        "asset": "did:::",
+        "": {},
+        "purpose": "",
+        "attachments": [],
+        "signatures": []
+      }
+    ],
+    "extensions": {}
+  }
+}
+```
+
+Key top-level fields include:
+* `specversion`: Specifies the version of the SACD specification being used. Currently, it is "1.0".
+* `timestamp`: The timestamp of when the SACD was created.
+* `type`: Identifies the document as a "dimo.sacd".
+* `data`: Contains the core information about the agreement.
+
+Within the `data` field, the following are typically found:
+* `grantor`: Details about the party granting access or providing the service. This includes their wallet address, an optional human-readable name, and additional information.
+* `grantee`: Details about the party receiving access or the service. Similar to the grantor, this includes their wallet address, an optional name, and additional information.
+* `effectiveAt`: The date and time when the agreement becomes active.
+* `expiresAt`: The date and time when the agreement expires.
+* `additionalDates`: A section to include any other relevant dates.
+* `agreement`: An array that can contain one or more specific agreement clauses, such as those related to payments or permissions. Each element in this array specifies a type of agreement and its details.
+* `extensions`: A section for adding any custom or non-standard fields relevant to a specific use case.
+
+### Use Case: Permissions
+
+When the type within the agreement array is set to "permission", the structure defines the access rights granted to a grantee for a specific asset. Relevant fields within this agreement type (as seen in [`sacd.permission.example.json`](data/sacd.permission.example.json)) include:
+
+* `asset`: A DID identifying the asset to which permissions are being granted, such as a specific NFT (e.g., did:nft:137:0x3330000000000000000000000000000000000000_31415).
+* `permissions`: An array detailing the specific permissions being granted. Each permission typically includes a name (e.g., "commands", "location:View:Current") and a human-readable description.
```

Comment:
It would be nice to nail down the format for these subobjects (permissions, attachments, signatures).